### ISSUE TITLE
Update ESLint packages

### ISSUE THREAD

**Issue by w3bdesign** [w3bdesign/stein-saks-papir#309]
@mrbrianevans

All the packages need to be updated. See existing pull requests for details.

Make sure there are no bugs after updating.

**Comment by w3bdesign** [comment]
@mrbrianevans

Note that I am using NPM version 7.

**Comment by mrbrianevans** [comment]
Okay will do. Please assign me to the issue

**Comment by w3bdesign** [comment]
@mrbrianevans 

Assigned.

**Comment by mrbrianevans** [comment]
@w3bdesign Can I remove this package? 
[`jest-environment-jsdom-sixteen`](https://www.npmjs.com/package/jest-environment-jsdom-sixteen)

I think it is redundant based on the message the author has put on it.

**Comment by w3bdesign** [comment]
@mrbrianevans 

If I remember correctly, it is used in package.json and  the following:

``` "test": "react-scripts test --watchAll=false --coverage --env=jest-environment-jsdom-sixteen", ``` 

As a result, I don't think removing it is a good idea, as it will cause the tests to fail.

**Comment by mrbrianevans** [comment]
Okay will do

**Comment by mrbrianevans** [comment]
Alright, I removed the package and the tests are still passing. 

Unrelated to that package, I have noticed some issues that might be worth creating GitHub issues for:
- when running the tests, even if they pass, there is an error message printed to the console:
``` console.error
      Warning: Can't perform a React state update on an unmounted component. This is a no-op, but it indicates a memory leak in your application. To fix, c
ancel all subscriptions and asynchronous tasks in a useEffect cleanup function.
          at Game (~\projects\stein-saks-papir\src\components\Pages\Index\Game.component.jsx:19:49)
```
(its visible in the output on the [circleCi log](https://app.circleci.com/pipelines/github/w3bdesign/stein-saks-papir/712/workflows/04544828-2bd0-4158-837c-4dbd89d96062/jobs/714/parallel-runs/0/steps/0-105))
- if you run the tests multiple times in a row, they pass most of the time, but sometimes randomly fail. There must be some race condition or randomness or something causing them to not be purely deterministic.
